source/apiVolontaria/apiVolontaria/views.py

```diff
@@ -95,6 +96,8 @@ class Users(generics.ListCreateAPIView):
     Create a new user.
     """
     serializer_class = serializers.UserBasicSerializer
+    filter_backends = (filters.SearchFilter,)
+    search_fields = ('^username', '^first_name', '^last_name', '^email')
```

Comment:
Why did you limit the utilisation to `startswith` ? 

It can be frustrating for users if they don't know exactly the begin of the field and it restrain a lit our capacity to search user with partial information.

Reply:
Okay, I'll remove this.